xbmc/storage/discs/AbstractDiscDriveHandler.h

```diff
@@ -0,0 +1,29 @@
+/*
+ *  Copyright (C) 2022 Team Kodi
+ *  This file is part of Kodi - https://kodi.tv
+ *
+ *  SPDX-License-Identifier: GPL-2.0-or-later
+ *  See LICENSES/README.md for more information.
+ */
+#pragma once
+
+#include "IDiscDriveHandler.h"
+
+#include <string>
+
+/*! \brief Implementation of an abstract DiscDriveHandler
+* \sa IDiscDriveHandler
+*/
+class AbstractDiscDriveHandler : public IDiscDriveHandler
```

Comment:
As this class can be instanciated, it does not have any pure virtual methids, the name `Abstract` is misleading. I would go with `CDiscDriveHandlerBase`instead.

Reply:
Nice point, maybe `DiscDriveHandlerBase` (the C is legacy/hungarian notation I guess)

Reply:
whoohoo